esrally/track/params.py

```diff
@@ -732,6 +735,7 @@ def __init__(
         self.on_conflict = on_conflict
         self.recency = recency
         self.pipeline = pipeline
+        self.refresh = refresh
```

Comment:
Noting that this initial review is just for the approach, but don't forget to add this param to the docstring for `PartitionBulkIndexParamSource`